GoTeamApp/GoTeamApp/TasksViewController.swift

```diff
@@ -18,13 +18,20 @@ class TasksViewController: UIViewController {
     @IBOutlet weak var addButton: UIButton!
     
     
+    var searchKey = ""
+
+    // cells
+    let kTaskCell = "TaskCell"
+    let kTaskWithAnnotationsCell = "TaskWithAnnotationsCell"
```

Comment:
Just FYI, don't know if this got back in here because of a rebase, but as a best practice and just to keep the code clean so we have a good portfolio, I have moved a lot of strings from view controllers to the 'Resources.swift' file, there are still some remaining, will move it as I see it.